src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -540,4 +540,12 @@ public void highlightKeyStringInMessageInfoHtml(){
         
     }
     
+    public boolean isTestingData() {
+        boolean isTestingAccount = false;
+        
+        if(email.endsWith(".tmt")){
+            isTestingAccount = true;
+        }
+        return isTestingAccount;
+    }
```

Comment:
You can simplify the function as
public boolean isTestingData() {
    return email.endsWith(".tmt");
}